string preset_name
---
moveit_msgs/AttachedCollisionObject attached_collision_object